app/views/journalUpload.html

```diff
@@ -288,9 +288,6 @@ <h4>
             <th>
               <a href="" translate>GL Account</a>
             </th>
-            <th>
-              <a href="" translate>Product Code</a>
```

Comment:
Another QA request. I was asked to remove this as some backend correction could happen in people soft that would not be reflected in the ERT. So instead of possibly showing an incorrect product code, we'll just remove the column.